src/components/VmsList/Pool.js

```diff
@@ -20,60 +16,53 @@ import { startPool } from '../../actions'
 import { getOsHumanName } from '../utils'
 
 /**
- * Single icon-card in the list
+ * Single icon-card in the list for a Pool
  */
-class Pool extends React.Component {
-  render () {
-    let { pool, icons, visibility, onStart } = this.props
-    const idPrefix = `pool-${pool.get('name')}`
-    const state = pool.get('status')
+const Pool = ({ pool, icons, onStart }) => {
+  const idPrefix = `pool-${pool.get('name')}`
+  const state = pool.get('status')
+  const osName = getOsHumanName(pool.getIn(['vm', 'os', 'type']))
+  const iconId = pool.getIn(['vm', 'icons', 'large', 'id'])
+  const icon = icons.get(iconId)
 
-    const iconId = pool.getIn(['vm', 'icons', 'large', 'id'])
-    const osName = getOsHumanName(pool.getIn(['vm', 'os', 'type']))
-    const icon = icons.get(iconId)
-    const isSelected = pool.get('id') === visibility.get('selectedPoolDetail')
-
-    return (
-      <div className={`col-xs-12 col-sm-6 col-md-4 col-lg-3 ${isSelected ? style['selectedPool'] : ''}`}>
-        <div className='card-pf card-pf-view card-pf-view-select card-pf-view-single-select'>
-          <div className='card-pf-body'>
-            <div className={`card-pf-top-element ${style['card-icon']}`}>
-              <Link to={`/pool/${pool.get('id')}`}>
-                <VmIcon icon={icon} className={style['card-pf-icon']}
-                  missingIconClassName='fa fa-birthday-cake card-pf-icon-circle' />
-              </Link>
-              <div>
-                <span className={style['operating-system-label']}>{ osName }</span>
-              </div>
+  return (
+    <div className={`col-xs-12 col-sm-6 col-md-4 col-lg-3`}>
```

Comment:
As patternfly-react's `Grid` component is introduced, this can be refactored as well. But within a follow-up.